### ISSUE TITLE
RvR flagged without taking an action

### ISSUE THREAD

**Issue by bwdaWAR** [WarEmu/WarBugs#7276]
I was on the Destruction side of the Eataine RvR lake with my Witch Hunter for some exploration, just after relogging to unflag after crossing the lake and moving further south. I encountered a Black Orc who attacked me, and to my surprise, I got RvR flagged without doing anything at all. As the combat log shows, I was hit and damaged several times before I realised that the completely unexpected actually happened and started attacking as well. I had no pets, auras, AoE tactics or anything of the kind (I had Brute Force, Flowing Accusation and Righteous Steel). As far as I know, this shouldn't be possible - that is, being RvR flagged, outside of a RvR area, without doing anything.
Here's a screenshot of the combat log:
![](http://i.imgur.com/RG1i3t2.jpg)

**Comment by Luth-RoR** [comment]
Are you 100% sure you have not been flagged earlier and maybe didn't notice it?
It could be that the RvR areas are not completely like they were on the mythic server and so you could have been flagged while walking through PvP.

**Comment by bwdaWAR** [comment]
I went back to that exact spot later and never had that countdown for being flagged when going into an RvR area appear. Also, the "You're now flagged" message and sound notification happened exactly when I saw the first attack animation from the Black Orc. As far as I recall, I either didn't touch the keyboard/mouse at all, or the chat text entry was active as I was either considering to or already typing. Since there is nothing in the combat log from me for the first 9s or so, I can be certain I didn't accidentally attack him.
